Regenerate Toscana dati from feed

joeyates opened the pull request:
This PR replaces the existing data for Tuscany with data taken from [OpenToscana's official feed](https://dati.toscana.it/dataset/rt-oraritb.xml).

Major changes:
* A number of bus companies in the previous file no longer exist and have been removed,
* All Tuscan bus routes are now covered,
* URLs are now all https,
* Geohashes are calculated to be as precise as possible while covering all stop locations.

The file is generated via this repo: https://github.com/joeyates/transitland-dati-toscana-it

drewda commented:
@joeyates thank you very much for all these updates.

Are any of the newly added feeds replacements for removed feeds? If so, please associate the old Onestop IDs using a `supersedes_ids` property.

Here is an example of how multiple outdated feeds were replaced by a single current feed record: https://github.com/transitland/transitland-atlas/blob/b3c3de9716d90c09420ecfcc4edff9d4d2075522/feeds/carrismetropolitana.github.com.dmfr.json#L6-L13

And here is an example of how an two operators were merged into one: https://github.com/transitland/transitland-atlas/blob/b3c3de9716d90c09420ecfcc4edff9d4d2075522/feeds/carrismetropolitana.github.com.dmfr.json#L41-L43

Thanks!

joeyates commented:
Hi @drewda 

I've added `supersedes_ids` where possible.

I've added back a couple of feeds that may be obsolete, but I think it's better that they stay in place until a new feed is available.

drewda commented:
Hi @joeyates great, thank you for your additions.

If any of these feeds become active again at new sources, we'll look forward to additional updates. Thanks.
